### ISSUE TITLE
colour errors different to standard output in the console?

### ISSUE THREAD

**Issue by geotheory** [dracula/rstudio#5]
Current implementation makes difficult to pick out errors in large process outputs in the console.

**Comment by kputschko** [comment]
Agreed, having regular messages be the same color as warnings and errors is challenging.
